src/index.js

```diff
@@ -1,10 +1,29 @@
 import './style.css';
 import { showMovies } from './modules/popullateMovie';
 import { showTvShows } from './modules/populateTvshows';
+import {
+  homeLinks, tvLinks, mainContainer, tvCardContainer, cardContainer,
+} from './modules/domElements';
 
 const component = () => {
   showMovies();
   showTvShows();
 };
 
-document.addEventListener('DOMContentLoaded', component);
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', component);
+
+homeLinks.addEventListener('click', () => {
+  tvCardContainer.classList.add('hide');
+  tvLinks.classList.remove('active');
+  homeLinks.classList.add('active');
+  mainContainer.classList.remove('hide');
+  cardContainer.classList.remove('hide');
```

Comment:
- [ ] The like functionality isn't consistent, Some times it increases one like and some times 4 likes. I would suggest you to fix this issue.

Reply:
Thanks update accordingly